tests/integration_tests/performance/test_snapshot_perf.py

```diff
@@ -42,6 +42,7 @@
     # We are also tracking restore from older version latency.
     # Snapshot properties: 2vCPU, 512MB RAM, 1 disk, 1 network iface.
     '0.23.0': 80,
```

Comment:
I think this also can be set to 8.

Reply:
Might be better to have a single latency here for all versions and have separate entries if we ever need to.

Reply:
Makes sense. Done